### ISSUE TITLE
ZFS inbuilt kernel fails to boot

### ISSUE THREAD

**Issue by tchalamarla** [zfsonlinux/zfs#7868]
### System information
Linux config
[zfsconfig.txt](https://github.com/zfsonlinux/zfs/files/2357744/zfsconfig.txt)

Type | Version/Name
 --- | --- 
Distribution Name	| 
Distribution Version	| 
Linux Kernel	| 4.14.y (LTS) also tried on 4.18.6
Architecture	| X86_64
ZFS Version	| master
SPL Version	| 

### Describe the problem you're observing
When booting a kernel with builtin ifs seeing kernel crash

### Describe how to reproduce the problem
in Linux
make defconfig
make prepare scripts

in ZFS
 ./configure  --enable-linux-builtin=yes --with-linux=${WORKSPACE}/linux -with-linux-obj=${WORKSPACE}/linux
 ./copy-builtin ${WORKSPACE}/linux/
In Linux build kernel with ZFS = Y

```
qemu-system-x86_64: warning: TCG doesn't support requested feature: CPUID.01H:ECX.vmx [bit 5]
[    0.000000] Linux version 4.18.6 (tirumalesh@tirumalesh) (gcc version 7.3.0 (Buildroot 2018.05.1-g4708797)) #1 SMP Thu Sep 6 07:39:33 PDT 2018
[    0.000000] Command line: console=ttyS0
[    0.000000] KERNEL supported cpus:
[    0.000000]   Intel GenuineIntel
[    0.000000]   AMD AuthenticAMD
[    0.000000]   Centaur CentaurHauls
[    0.000000] x86/fpu: x87 FPU will use FXSAVE
[    0.000000] BIOS-provided physical RAM map:
[    0.000000] BIOS-e820: [mem 0x0000000000000000-0x000000000009fbff] usable
[    0.000000] BIOS-e820: [mem 0x000000000009fc00-0x000000000009ffff] reserved
[    0.000000] BIOS-e820: [mem 0x00000000000f0000-0x00000000000fffff] reserved
[    0.000000] BIOS-e820: [mem 0x0000000000100000-0x0000000007fddfff] usable
[    0.000000] BIOS-e820: [mem 0x0000000007fde000-0x0000000007ffffff] reserved
[    0.000000] BIOS-e820: [mem 0x00000000fffc0000-0x00000000ffffffff] reserved
[    0.000000] NX (Execute Disable) protection: active
[    0.000000] SMBIOS 2.8 present.
[    0.000000] DMI: QEMU Standard PC (i440FX + PIIX, 1996), BIOS 1.10.2-1ubuntu1 04/01/2014
[    0.000000] last_pfn = 0x7fde max_arch_pfn = 0x400000000
[    0.000000] x86/PAT: Configuration [0-7]: WB  WC  UC- UC  WB  WP  UC- WT  
[    0.000000] found SMP MP-table at [mem 0x000f6a90-0x000f6a9f] mapped at [(____ptrval____)]
[    0.000000] Scanning 1 areas for low memory corruption
[    0.000000] ACPI: Early table checksum verification disabled
[    0.000000] ACPI: RSDP 0x00000000000F68C0 000014 (v00 BOCHS )
[    0.000000] ACPI: RSDT 0x0000000007FE15FC 000030 (v01 BOCHS  BXPCRSDT 00000001 BXPC 00000001)
[    0.000000] ACPI: FACP 0x0000000007FE1458 000074 (v01 BOCHS  BXPCFACP 00000001 BXPC 00000001)
[    0.000000] ACPI: DSDT 0x0000000007FE0040 001418 (v01 BOCHS  BXPCDSDT 00000001 BXPC 00000001)
[    0.000000] ACPI: FACS 0x0000000007FE0000 000040
[    0.000000] ACPI: APIC 0x0000000007FE154C 000078 (v01 BOCHS  BXPCAPIC 00000001 BXPC 00000001)
[    0.000000] ACPI: HPET 0x0000000007FE15C4 000038 (v01 BOCHS  BXPCHPET 00000001 BXPC 00000001)
[    0.000000] No NUMA configuration found
[    0.000000] Faking a node at [mem 0x0000000000000000-0x0000000007fddfff]
[    0.000000] NODE_DATA(0) allocated [mem 0x07fb3000-0x07fddfff]
[    0.000000] tsc: Fast TSC calibration using PIT
[    0.000000] Zone ranges:
[    0.000000]   DMA      [mem 0x0000000000001000-0x0000000000ffffff]
[    0.000000]   DMA32    [mem 0x0000000001000000-0x0000000007fddfff]
[    0.000000]   Normal   empty
[    0.000000]   Device   empty
[    0.000000] Movable zone start for each node
[    0.000000] Early memory node ranges
[    0.000000]   node   0: [mem 0x0000000000001000-0x000000000009efff]
[    0.000000]   node   0: [mem 0x0000000000100000-0x0000000007fddfff]
[    0.000000] Reserved but unavailable: 132 pages
[    0.000000] Initmem setup node 0 [mem 0x0000000000001000-0x0000000007fddfff]
[    0.000000] ACPI: PM-Timer IO Port: 0x608
[    0.000000] ACPI: LAPIC_NMI (acpi_id[0xff] dfl dfl lint[0x1])
[    0.000000] IOAPIC[0]: apic_id 0, version 32, address 0xfec00000, GSI 0-23
[    0.000000] ACPI: INT_SRC_OVR (bus 0 bus_irq 0 global_irq 2 dfl dfl)
[    0.000000] ACPI: INT_SRC_OVR (bus 0 bus_irq 5 global_irq 5 high level)
[    0.000000] ACPI: INT_SRC_OVR (bus 0 bus_irq 9 global_irq 9 high level)
[    0.000000] ACPI: INT_SRC_OVR (bus 0 bus_irq 10 global_irq 10 high level)
[    0.000000] ACPI: INT_SRC_OVR (bus 0 bus_irq 11 global_irq 11 high level)
[    0.000000] Using ACPI (MADT) for SMP configuration information
[    0.000000] ACPI: HPET id: 0x8086a201 base: 0xfed00000
[    0.000000] smpboot: Allowing 1 CPUs, 0 hotplug CPUs
[    0.000000] PM: Registered nosave memory: [mem 0x00000000-0x00000fff]
[    0.000000] PM: Registered nosave memory: [mem 0x0009f000-0x0009ffff]
[    0.000000] PM: Registered nosave memory: [mem 0x000a0000-0x000effff]
[    0.000000] PM: Registered nosave memory: [mem 0x000f0000-0x000fffff]
[    0.000000] [mem 0x08000000-0xfffbffff] available for PCI devices
[    0.000000] Booting paravirtualized kernel on bare hardware
[    0.000000] clocksource: refined-jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 7645519600211568 ns
[    0.000000] random: get_random_bytes called from start_kernel+0x99/0x55a with crng_init=0
[    0.000000] setup_percpu: NR_CPUS:8192 nr_cpumask_bits:1 nr_cpu_ids:1 nr_node_ids:1
[    0.000000] percpu: Embedded 46 pages/cpu @(____ptrval____) s151552 r8192 d28672 u2097152
[    0.000000] Built 1 zonelists, mobility grouping on.  Total pages: 32103
[    0.000000] Policy zone: DMA32
[    0.000000] Kernel command line: console=ttyS0
[    0.000000] Memory: 96132K/130544K available (14348K kernel code, 2750K rwdata, 5172K rodata, 2516K init, 3892K bss, 34412K reserved, 0K cma-reserved)
[    0.000000] SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
[    0.000000] ftrace: allocating 46225 entries in 181 pages
[    0.000000] Hierarchical RCU implementation.
[    0.000000] 	RCU restricting CPUs from NR_CPUS=8192 to nr_cpu_ids=1.
[    0.000000] 	Tasks RCU enabled.
[    0.000000] RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=1
[    0.000000] NR_IRQS: 524544, nr_irqs: 256, preallocated irqs: 16
[    0.000000] Console: colour VGA+ 80x25
[    0.000000] console [ttyS0] enabled
[    0.000000] ACPI: Core revision 20180531
[    0.000000] clocksource: hpet: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 19112604467 ns
[    0.004000] APIC: Switch to symmetric I/O mode setup
[    0.008000] ..TIMER: vector=0x30 apic1=0 pin1=2 apic2=-1 pin2=-1
[    0.028000] tsc: Fast TSC calibration using PIT
[    0.028000] tsc: Detected 3191.978 MHz processor
[    0.028000] clocksource: tsc-early: mask: 0xffffffffffffffff max_cycles: 0x2e02aff4ae6, max_idle_ns: 440795229291 ns
[    0.032279] Calibrating delay loop (skipped), value calculated using timer frequency.. 6383.95 BogoMIPS (lpj=12767912)
[    0.032723] pid_max: default: 32768 minimum: 301
[    0.033580] Security Framework initialized
[    0.033832] Yama: becoming mindful.
[    0.035550] AppArmor: AppArmor initialized
[    0.037593] Dentry cache hash table entries: 16384 (order: 5, 131072 bytes)
[    0.037900] Inode-cache hash table entries: 8192 (order: 4, 65536 bytes)
[    0.038320] Mount-cache hash table entries: 512 (order: 0, 4096 bytes)
[    0.038604] Mountpoint-cache hash table entries: 512 (order: 0, 4096 bytes)
[    0.049069] mce: CPU supports 10 MCE banks
[    0.049894] Last level iTLB entries: 4KB 0, 2MB 0, 4MB 0
[    0.050058] Last level dTLB entries: 4KB 0, 2MB 0, 4MB 0, 1GB 0
[    0.050366] Spectre V2 : Spectre mitigation: LFENCE not serializing, switching to generic retpoline
[    0.050710] Spectre V2 : Mitigation: Full generic retpoline
[    0.050922] Spectre V2 : Spectre v2 / SpectreRSB mitigation: Filling RSB on context switch
[    0.051203] Speculative Store Bypass: Vulnerable
[    0.365648] Freeing SMP alternatives memory: 40K
[    0.384000] smpboot: CPU0: AMD QEMU Virtual CPU version 2.5+ (family: 0x6, model: 0x6, stepping: 0x3)
[    0.384000] Performance Events: PMU not available due to virtualization, using software events only.
[    0.385483] Hierarchical SRCU implementation.
[    0.391201] NMI watchdog: Perf event create on CPU 0 failed with -2
[    0.391364] NMI watchdog: Perf NMI watchdog permanently disabled
[    0.393794] smp: Bringing up secondary CPUs ...
[    0.394014] smp: Brought up 1 node, 1 CPU
[    0.394168] smpboot: Max logical packages: 1
[    0.394354] smpboot: Total of 1 processors activated (6383.95 BogoMIPS)
[    0.403607] devtmpfs: initialized
[    0.405928] x86/mm: Memory block size: 128MB
[    0.409789] clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 7645041785100000 ns
[    0.410311] futex hash table entries: 256 (order: 2, 16384 bytes)
[    0.412225] pinctrl core: initialized pinctrl subsystem
[    0.417836] RTC time: 15:22:03, date: 09/06/18
[    0.422188] NET: Registered protocol family 16
[    0.424299] audit: initializing netlink subsys (disabled)
[    0.426985] cpuidle: using governor ladder
[    0.427232] cpuidle: using governor menu
[    0.428423] audit: type=2000 audit(1536247322.424:1): state=initialized audit_enabled=0 res=1
[    0.428926] ACPI: bus type PCI registered
[    0.429168] acpiphp: ACPI Hot Plug PCI Controller Driver version: 0.5
[    0.430972] PCI: Using configuration type 1 for base access
[    0.516801] HugeTLB registered 2.00 MiB page size, pre-allocated 0 pages
[    0.520917] ACPI: Added _OSI(Module Device)
[    0.521080] ACPI: Added _OSI(Processor Device)
[    0.521221] ACPI: Added _OSI(3.0 _SCP Extensions)
[    0.521389] ACPI: Added _OSI(Processor Aggregator Device)
[    0.521609] ACPI: Added _OSI(Linux-Dell-Video)
[    0.539895] ACPI: 1 ACPI AML tables successfully acquired and loaded
[    0.553918] ACPI: Interpreter enabled
[    0.554718] ACPI: (supports S0 S3 S4 S5)
[    0.554905] ACPI: Using IOAPIC for interrupt routing
[    0.555514] PCI: Using host bridge windows from ACPI; if necessary, use "pci=nocrs" and report a bug
[    0.557777] ACPI: Enabled 2 GPEs in block 00 to 0F
[    0.597151] ACPI: PCI Root Bridge [PCI0] (domain 0000 [bus 00-ff])
[    0.597653] acpi PNP0A03:00: _OSC: OS supports [ASPM ClockPM Segments MSI]
[    0.598093] acpi PNP0A03:00: _OSC failed (AE_NOT_FOUND); disabling ASPM
[    0.598724] acpi PNP0A03:00: fail to add MMCONFIG information, can't access extended PCI configuration space under this bridge.
[    0.604114] acpiphp: Slot [3] registered
[    0.604489] acpiphp: Slot [4] registered
[    0.604803] acpiphp: Slot [5] registered
[    0.605120] acpiphp: Slot [6] registered
[    0.605417] acpiphp: Slot [7] registered
[    0.605707] acpiphp: Slot [8] registered
[    0.606014] acpiphp: Slot [9] registered
[    0.606348] acpiphp: Slot [10] registered
[    0.606669] acpiphp: Slot [11] registered
[    0.606975] acpiphp: Slot [12] registered
[    0.607288] acpiphp: Slot [13] registered
[    0.607571] acpiphp: Slot [14] registered
[    0.608176] acpiphp: Slot [15] registered
[    0.608615] acpiphp: Slot [16] registered
[    0.608982] acpiphp: Slot [17] registered
[    0.609306] acpiphp: Slot [18] registered
[    0.609575] acpiphp: Slot [19] registered
[    0.609861] acpiphp: Slot [20] registered
[    0.610179] acpiphp: Slot [21] registered
[    0.610478] acpiphp: Slot [22] registered
[    0.610790] acpiphp: Slot [23] registered
[    0.611120] acpiphp: Slot [24] registered
[    0.611440] acpiphp: Slot [25] registered
[    0.611763] acpiphp: Slot [26] registered
[    0.612189] acpiphp: Slot [27] registered
[    0.612524] acpiphp: Slot [28] registered
[    0.612818] acpiphp: Slot [29] registered
[    0.613113] acpiphp: Slot [30] registered
[    0.613444] acpiphp: Slot [31] registered
[    0.613898] PCI host bridge to bus 0000:00
[    0.614155] pci_bus 0000:00: root bus resource [io  0x0000-0x0cf7 window]
[    0.614443] pci_bus 0000:00: root bus resource [io  0x0d00-0xffff window]
[    0.614704] pci_bus 0000:00: root bus resource [mem 0x000a0000-0x000bffff window]
[    0.615024] pci_bus 0000:00: root bus resource [mem 0x08000000-0xfebfffff window]
[    0.615345] pci_bus 0000:00: root bus resource [mem 0x100000000-0x17fffffff window]
[    0.615677] pci_bus 0000:00: root bus resource [bus 00-ff]
[    0.622886] pci 0000:00:01.1: legacy IDE quirk: reg 0x10: [io  0x01f0-0x01f7]
[    0.623083] pci 0000:00:01.1: legacy IDE quirk: reg 0x14: [io  0x03f6]
[    0.623326] pci 0000:00:01.1: legacy IDE quirk: reg 0x18: [io  0x0170-0x0177]
[    0.624058] pci 0000:00:01.1: legacy IDE quirk: reg 0x1c: [io  0x0376]
[    0.625598] pci 0000:00:01.3: quirk: [io  0x0600-0x063f] claimed by PIIX4 ACPI
[    0.625776] pci 0000:00:01.3: quirk: [io  0x0700-0x070f] claimed by PIIX4 SMB
[    0.652941] ACPI: PCI Interrupt Link [LNKA] (IRQs 5 *10 11)
[    0.654137] ACPI: PCI Interrupt Link [LNKB] (IRQs 5 *10 11)
[    0.655211] ACPI: PCI Interrupt Link [LNKC] (IRQs 5 10 *11)
[    0.656867] ACPI: PCI Interrupt Link [LNKD] (IRQs 5 10 *11)
[    0.657546] ACPI: PCI Interrupt Link [LNKS] (IRQs *9)
[    0.661971] pci 0000:00:02.0: vgaarb: setting as boot VGA device
[    0.662251] pci 0000:00:02.0: vgaarb: VGA device added: decodes=io+mem,owns=io+mem,locks=none
[    0.662615] pci 0000:00:02.0: vgaarb: bridge control possible
[    0.662865] vgaarb: loaded
[    0.666035] SCSI subsystem initialized
[    0.667123] ACPI: bus type USB registered
[    0.668266] usbcore: registered new interface driver usbfs
[    0.668669] usbcore: registered new interface driver hub
[    0.669143] usbcore: registered new device driver usb
[    0.669680] pps_core: LinuxPPS API ver. 1 registered
[    0.669919] pps_core: Software ver. 5.3.6 - Copyright 2005-2007 Rodolfo Giometti <giometti@linux.it>
[    0.670282] PTP clock support registered
[    0.671001] EDAC MC: Ver: 3.0.0
[    0.673389] PCI: Using ACPI for IRQ routing
[    0.678744] NetLabel: Initializing
[    0.678882] NetLabel:  domain hash size = 128
[    0.679061] NetLabel:  protocols = UNLABELED CIPSOv4 CALIPSO
[    0.680000] NetLabel:  unlabeled traffic allowed by default
[    0.682084] HPET: 3 timers in total, 0 timers will be used for per-cpu timer
[    0.682388] hpet0: at MMIO 0xfed00000, IRQs 2, 8, 0
[    0.682634] hpet0: 3 comparators, 64-bit 100.000000 MHz counter
[    0.688364] clocksource: Switched to clocksource tsc-early
[    0.784592] VFS: Disk quotas dquot_6.6.0
[    0.784925] VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
[    0.788951] AppArmor: AppArmor Filesystem Enabled
[    0.789582] pnp: PnP ACPI init
[    0.795340] pnp: PnP ACPI: found 6 devices
[    0.816269] clocksource: acpi_pm: mask: 0xffffff max_cycles: 0xffffff, max_idle_ns: 2085701024 ns
[    0.817949] NET: Registered protocol family 2
[    0.822211] tcp_listen_portaddr_hash hash table entries: 256 (order: 0, 4096 bytes)
[    0.822530] TCP established hash table entries: 1024 (order: 1, 8192 bytes)
[    0.822881] TCP bind hash table entries: 1024 (order: 2, 16384 bytes)
[    0.823175] TCP: Hash tables configured (established 1024 bind 1024)
[    0.824573] UDP hash table entries: 256 (order: 1, 8192 bytes)
[    0.824891] UDP-Lite hash table entries: 256 (order: 1, 8192 bytes)
[    0.826267] NET: Registered protocol family 1
[    0.826657] pci 0000:00:00.0: Limiting direct PCI/PCI transfers
[    0.826841] pci 0000:00:01.0: PIIX3: Enabling Passive Release
[    0.827180] pci 0000:00:01.0: Activating ISA DMA hang workarounds
[    0.827740] pci 0000:00:02.0: Video device with shadowed ROM at [mem 0x000c0000-0x000dffff]
[    0.832284] Scanning for low memory corruption every 60 seconds
[    0.836782] Initialise system trusted keyrings
[    0.837973] Key type blacklist registered
[    0.838713] workingset: timestamp_bits=36 max_order=15 bucket_order=0
[    0.847588] zbud: loaded
[    1.857281] tsc: Refined TSC clocksource calibration: 3192.019 MHz
[    1.857618] clocksource: tsc: mask: 0xffffffffffffffff max_cycles: 0x2e02d6b8161, max_idle_ns: 440795336256 ns
[    1.858039] clocksource: Switched to clocksource tsc
[    2.499516] BUG: unable to handle kernel NULL pointer dereference at 000000000000008c
[    2.499740] PGD 0 P4D 0 
[    2.499912] Oops: 0000 [#1] SMP NOPTI
[    2.500208] CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.18.6 #1
[    2.500408] Hardware name: QEMU Standard PC (i440FX + PIIX, 1996), BIOS 1.10.2-1ubuntu1 04/01/2014
[    2.500777] RIP: 0010:kobject_get+0x10/0x50
[    2.500777] Code: ff eb a4 48 c7 c2 f8 b8 a7 8b eb cb 0f 1f 40 00 66 2e 0f 1f 84 00 00 00 00 00 48 85 ff 48 89 f8 74 38 55 48 89 e5 48 83 ec 08 <f6> 47 3c 01 74 0c 3e ff 40 38 0f 88 4d 3f 01 00 c9 c3 48 8b 37 48 
[    2.500777] RSP: 0018:ffffb3e200043ce8 EFLAGS: 00000296
[    2.500777] RAX: 0000000000000050 RBX: ffffffff8c4265e0 RCX: 0000000000000000
[    2.500777] RDX: 000000000000002f RSI: 000000000000002f RDI: 0000000000000050
[    2.500777] RBP: ffffb3e200043cf0 R08: 0000000000000000 R09: ffffa06c07001c00
[    2.500777] R10: ffffb3e200043dc0 R11: 00000000000002b0 R12: 0000000000000000
[    2.500777] R13: 0000000000000050 R14: 0000000000000007 R15: ffffffff8beea91e
[    2.500777] FS:  0000000000000000(0000) GS:ffffa06c07c00000(0000) knlGS:0000000000000000
[    2.500777] CS:  0010 DS: 0000 ES: 0000 CR0: 0000000080050033
[    2.500777] CR2: 000000000000008c CR3: 000000000520a000 CR4: 00000000000006f0
[    2.500777] Call Trace:
[    2.500777]  kobject_add_internal+0x3b/0x2b0
[    2.500777]  ? kfree_const+0x20/0x30
[    2.500777]  ? set_debug_rodata+0x17/0x17
[    2.500777]  kobject_add+0x71/0xd0
[    2.500777]  ? zfs_sa_upgrade_txholds+0x80/0x80
[    2.500777]  ? trace_event_define_fields_zfs_arc_miss_class+0x4a7/0x4a7
[    2.500777]  zfs_sysfs_init+0x63/0x210
[    2.500777]  ? trace_event_define_fields_zfs_arc_miss_class+0x4a7/0x4a7
[    2.500777]  ? set_debug_rodata+0x17/0x17
[    2.500777]  _init+0x147a/0x15c1
[    2.500777]  do_one_initcall+0x4a/0x1c9
[    2.500777]  kernel_init_freeable+0x1c5/0x271
[    2.500777]  ? rest_init+0xb0/0xb0
[    2.500777]  kernel_init+0xe/0x110
[    2.500777]  ret_from_fork+0x35/0x40
[    2.500777] Modules linked in:
[    2.500777] CR2: 000000000000008c
[    2.500777] ---[ end trace 976e727b87365e09 ]---
[    2.500777] RIP: 0010:kobject_get+0x10/0x50
[    2.500777] Code: ff eb a4 48 c7 c2 f8 b8 a7 8b eb cb 0f 1f 40 00 66 2e 0f 1f 84 00 00 00 00 00 48 85 ff 48 89 f8 74 38 55 48 89 e5 48 83 ec 08 <f6> 47 3c 01 74 0c 3e ff 40 38 0f 88 4d 3f 01 00 c9 c3 48 8b 37 48 
[    2.500777] RSP: 0018:ffffb3e200043ce8 EFLAGS: 00000296
[    2.500777] RAX: 0000000000000050 RBX: ffffffff8c4265e0 RCX: 0000000000000000
[    2.500777] RDX: 000000000000002f RSI: 000000000000002f RDI: 0000000000000050
[    2.500777] RBP: ffffb3e200043cf0 R08: 0000000000000000 R09: ffffa06c07001c00
[    2.500777] R10: ffffb3e200043dc0 R11: 00000000000002b0 R12: 0000000000000000
[    2.500777] R13: 0000000000000050 R14: 0000000000000007 R15: ffffffff8beea91e
[    2.500777] FS:  0000000000000000(0000) GS:ffffa06c07c00000(0000) knlGS:0000000000000000
[    2.500777] CS:  0010 DS: 0000 ES: 0000 CR0: 0000000080050033
[    2.500777] CR2: 000000000000008c CR3: 000000000520a000 CR4: 00000000000006f0
[    2.513937] Kernel panic - not syncing: Attempted to kill init! exitcode=0x00000009
[    2.513937] 
[    2.514447] Kernel Offset: 0x9600000 from 0xffffffff81000000 (relocation range: 0xffffffff80000000-0xffffffffbfffffff)
[    2.514886] ---[ end Kernel panic - not syncing: Attempted to kill init! exitcode=0x00000009

```

**Comment by tchalamarla** [comment]
I tried it, it fails to even compile complaints about some gnu problem

**Comment by behlendorf** [comment]
@tchalamarla would you mind testing the fix in #7872.

**Comment by tchalamarla** [comment]
Will do and update
